chromium.yaml

```diff
@@ -252,6 +252,8 @@ pipeline:
       # We could parse clang --version with awk, but this is cleaner
       clang_version=$(basename $(clang --print-resource-dir))
 
+      sed -i 's/--warning-suppression-mappings=/-Wunused --warning-suppression-mappings=/g' ~/build/config/compiler/BUILD.gn
+
```

Comment:
Rather than disabling using sed can't we just pass `clang_warning_suppression=false` as part of the `--args` below to disable the addition of this flag?  I might be misunderstanding the way the build system works.

Reply:
args vs config - I just finished reading that bit of the built tool docs.